pages/index.tsx

```diff
@@ -42,8 +43,11 @@ export default function Home({ feed }) {
 }
 
 export async function getServerSideProps() {
-  const res = await fetch('http://localhost:3000/api/feed')
-  const feed = await res.json()
+  const feed = await prisma.story.findMany({
+    where: { approved: true },
+    orderBy: { createdAt: 'asc' },
+  })
+
```

Comment:
Get data directly from the DB rather than through the API route.

Reply:
How does the frontend have access to the DB?